config/configuration.json

```diff
@@ -36,6 +36,7 @@
     }
   },
   "BuildingRoadIntersectionCheck": {
+    "highway.filter": "highway->motorway_link,primary_link,primary,residential,secondary_link,secondary,service,tertiary_link,tertiary,track,trunk_link,trunk,unclassified,motorway,road",
```

Comment:
Is there a specific reason for adding a highway tag filter? If the goal was to filter non navigable roads, couldn't you just add a test (on the enum set) in the `validCheckForObject` class? Im curious if theres a use case where we'd actually want to update this. 
@alxswny

Reply:
@danielduhh I thought leaving it as a configurable would be good for the future so that it could be run to include all highways for specific cases.